source/CommonJobs/CommonJobs.MVC.UI/Scripts/evaluations-dashboard.js

```diff
@@ -183,8 +191,39 @@
         this.evaluatorsAmount = ko.computed(function () {
             return this.evaluators.length;
         }, this);
+        this.evaluatorsTextLink = ko.computed(function () {
+            return (this.evaluatorsAmount() === 1) ? this.evaluatorsAmount() + " calificador" : this.evaluatorsAmount() + " calificadores";
+        }, this);
         this.state = data.State;
-        this.stateName = evaluationStates[data.State];
+        this.stateName = evaluationStates[this.state];
+        this.stateClasses = "state-doc state-" + this.state;
+        this.isCalificatorsEditable = ko.computed(function () {
+            return this.isResponsible && this.state != 6;
+        }, this);
+        this.calificationActionTooltip = ko.observable('');
+        this.calificationActionText = ko.observable('');
+        this.calificationActionClass = ko.observable('');
+        this.calificationAction = ko.computed(function () {
+            switch (this.state) {
+                case 0:
+                case 2:
+                    this.calificationActionTooltip("Calificar como responsable");
```

Comment:
I think that it is not a good idea to have collateral effects on computed evaluation. Maybe it is better to have a subscription, or better yet, multiple computed properties.